### PR TITLE
create a catalog source and test it

### PR DESCRIPTION
**Pull request by adtandale**

### CODE REVIEW

**Comment by adtandale** [comment]
Please review.

File: piqe_ocp_lib/api/resources/ocp_operators.py [hunk 1]
```diff
@@ -306,6 +306,24 @@ def __init__(self, kube_config_file):
         self.kind = "CatalogSource"
         self.catalog_source_obj = self.dyn_client.resources.get(api_version=self.api_version, kind=self.kind)
 
+    def create_catalog_source(self, cs_name, image, namespace="openshift-marketplace"):
+        spec_dict =  {"displayName": "Optional operators",  "icon": {"base64data": "", "mediatype": ""} ,
```
Comment (btjd) [inline piqe_ocp_lib/api/resources/ocp_operators.py:310]: If spec_dict is not going to be prameterized, then there is no benefit in separating the two. It looks like for this method the only two required arguments are cs_name and image, in that case just put the whole dict together under one object. Maybe make displayName and publisher as optional arguments and give them the currently assigned values by default.

**Review by btjd — CHANGES_REQUESTED** [review]

File: piqe_ocp_lib/api/resources/ocp_operators.py (hunk 1 above)
Comment (adtandale) [reply piqe_ocp_lib/api/resources/ocp_operators.py]: Done, please check.

**Comment by Dannyb48** [comment]
@adtandale Can you please make sure you have the latest changes from master? I saw the downstream CI failed but not the typical 3 tests. There were more failures that were fixed with recent changes.

**Comment by adtandale** [comment]
> @adtandale Can you please make sure you have the latest changes from master? I saw the downstream CI failed but not the typical 3 tests. There were more failures that were fixed with recent changes.

@Dannyb48  thanks for suggesting, I have added now and submitted the changes again, can you please check.

**Review by btjd — APPROVED** [review]

**Review by Dannyb48 — APPROVED** [review]

**Pull request merged**